src/ecommerce.js

```diff
@@ -103,8 +105,11 @@ export default function Ecommerce(mpInstance) {
                 return Types.CommerceEventType.ProductRemoveFromCart;
             case Types.ProductActionType.RemoveFromWishlist:
                 return Types.CommerceEventType.ProductRemoveFromWishlist;
+
+            // https://go.mparticle.com/work/SQDSDKS-4801
             case Types.ProductActionType.Unknown:
                 return Types.EventType.Unknown;
+
```

Comment:
```suggestion
```

Reply:
The space is intentional to show that the specific case relates to the worklink above it. IMO it makes it easier to read and highlights the relationship between the case and the worklink. I can remove if you feel strongly about it.